doc/concepts/recipe.rst

```diff
@@ -38,7 +39,8 @@ The steps of building a simulation from a recipe are:
 
 .. topic:: 1. Load balancing
 
-    First, the cells are partitioned over MPI ranks, and each rank parses
+    This can be done manually, by creating a :ref:`domain decomposition<modeldomdec>`
+    object. Or by using an Arbor-provided load balancer. First, the cells are partitioned over MPI ranks, and each rank parses
```

Comment:
You *either* use domain decomp *or* the Arbor load balancer, which is what is explained here, right? Maybe start the section with sth like "There are two ways to balance load, 1. domain decomp and 2. use an Arbor-provided balancer. This automatic balancer works thus: ..."

Reply:
Well The arbor-provided load balancer will generate the domain decomposition object. So how about: 
```
This can be done manually by the user, through creating a domain decomposition object; 
or automatically by Arbor using a load balancer that generates the domain decomposition 
object according to some heuristics. 
```
I don't plan on going into too much detail about this in this PR. I can add more details to the domain decomposition page when I get to it later, then I can modify this page.

Reply:
OK!

Reply:
I'm actually going to undo these changes because I misread the section underneath. They don't make much sense here.